index.html

```diff
@@ -202,82 +181,60 @@ <h1>Prototype Viewer MediaWiki 3D GLTF Data, STL data is already available </h1>
             const environment = new THREE.RoomEnvironment(this.renderer);
             const pmremGenerator = new THREE.PMREMGenerator(this.renderer);
             this.scene = new THREE.Scene();
-            //this.scene.background = new THREE.Color(0xbbbbbb);
-            this.scene.background = new THREE.Color(0x0000000);
```

Comment:
Note that `scene.background` and `renderer.setClearColor` effectively do the same thing for our purposes – no need to specify both.